mail_activity_done/migrations/18.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,11 @@
+from odoo.addons.mail_activity_done.hooks import _set_keep_done
+
+
+def migrate(cr, version):
+    """Set keep_done on all existing activity types"""
+    _set_keep_done(cr)
+    cr.execute(
+        """
+        delete from mail_activity where active is not true and done is not true;
```

Comment:
this one is unexpected for me, what's your rationale here?

Reply:
All archived activites are considered done in Odoo 18: https://github.com/odoo/odoo/blob/44e61c6/addons/mail/models/mail_activity.py#L154

Reply:
yes but we're migrating the module that people install to keep activities, so I'd not expect its migration to delete anything. I'd expect the migration script to set active to False where done is True (even though the v17 code of this module should have set this in the first place)

Reply:
True, handling done activities that are still active is another inconsistency that might need handling. As for the other case, I downgraded that to a warning.